### PR TITLE
Update the config to delete the recorded video if no tests are retried

### PR DESCRIPTION
**Pull request by shyambh**
I have updated the cypress.config.js file to add the code to delete the recorded videos if none of the tests are retried.
I basically simply followed the steps mentioned here:
https://docs.cypress.io/api/plugins/after-spec-api#Delete-the-recorded-video-if-no-tests-retried

### CODE REVIEW

File: cypress.config.js [hunk 1]
```diff
@@ -1,7 +1,8 @@
-const {defineConfig} = require('cypress');
-const {tagify} = require('cypress-tags');
+const { defineConfig } = require('cypress');
```
Comment (peterjaap) [inline cypress.config.js:1]: Could you remove the spaces here? The coding standard requires there are no spaces there.

File: cypress.config.js [hunk 2]
```diff
@@ -1,7 +1,8 @@
...
+const { tagify } = require('cypress-tags');
```
Comment (peterjaap) [inline cypress.config.js:2]: Could you remove the spaces here? The coding standard requires there are no spaces there.

Comment (shyambh) [reply cypress.config.js]: Hi @peterjaap , can you please take a look now? I have removed the whitespaces.

**Review by peterjaap — APPROVED** [review]

**Comment by shyambh** [comment]
Also can you please add 'hactoberfest-accepted' label to this PR if you think it is fine?

Like so:
image

**Comment by peterjaap** [comment]
@shyambh I didn't know I had to do that, sorry. Done now.

**Comment by Vinai** [comment]
This MR breaks the config with
```
Your configFile is invalid: ...magento2-cypress-testing-suite/cypress.config.js

It threw an error when required, check the stack trace below:

Error [ERR_REQUIRE_ESM]: require() of ES Module ...magento2-cypress-testing-suite/node_modules/del/index.js from ...magento2-cypress-testing-suite/cypress.config.js not supported.
Instead change the require of index.js in /Volumes/CaseSensitive/Workspace/php-sites/mos-hyva-245/magento2-cypress-testing-suite/cypress.config.js to a dynamic import() which is available in all CommonJS modules.
```

**Comment by Vinai** [comment]
Adding a dependency for something as simple as deleting files and folders recursively is maybe not necessary.

**Comment by shyambh** [comment]
@Vinai Thank you for this. I will take a look at this.

**Comment by Vinai** [comment]
Did you even test the code before submitting the PR @shyambh ?

**Comment by Vinai** [comment]
Don't worry, already opened a PR https://github.com/elgentos/magento2-cypress-testing-suite/pull/100

**Pull request merged**